Files/UserControls/Selection/GenericSelectedItems.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
+namespace Files.UserControls.Selection
+{
+    public class GenericSelectedItems : ISelectedItems
```

Comment:
Same comment as ISelectedItems

Reply:
If there is a better solution from the problem above I will happily get rid of this :blush: